From bd90c1ffbdcacb1cfd8e685ed46e8fd129775213 Mon Sep 17 00:00:00 2001
From: Chenhui Zhao <chenhui.zhao@nxp.com>
Date: Fri, 22 Jan 2016 18:15:30 +0800
Subject: [PATCH 043/752] powerpc/dts: fix dts related problems

* add the SCFG node to t1040si-post.dtsi
* change t1023 to t1024
* change "fsl,qoriq-rcpm-2.0" to "fsl,qoriq-rcpm-2.1" in t1023si-post.dtsi

Signed-off-by: Chenhui Zhao <chenhui.zhao@nxp.com>
[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/powerpc/platforms/85xx/deepsleep.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/powerpc/platforms/85xx/deepsleep.c b/arch/powerpc/platforms/85xx/deepsleep.c
index ef10a3b..950e585 100644
--- a/arch/powerpc/platforms/85xx/deepsleep.c
+++ b/arch/powerpc/platforms/85xx/deepsleep.c
@@ -293,7 +293,7 @@ static int __init fsl_dp_iomap(void)
 
 	fsl_dp_priv.ccsr_scfg_base = fsl_of_iomap("fsl,t1040-scfg");
 	if (!fsl_dp_priv.ccsr_scfg_base) {
-		fsl_dp_priv.ccsr_scfg_base = fsl_of_iomap("fsl,t1023-scfg");
+		fsl_dp_priv.ccsr_scfg_base = fsl_of_iomap("fsl,t1024-scfg");
 		if (!fsl_dp_priv.ccsr_scfg_base)
 			goto err;
 	}
@@ -326,7 +326,7 @@ static int __init fsl_dp_iomap(void)
 			fsl_of_iomap("fsl,t1040-l3-cache-controller");
 	if (!fsl_dp_priv.ccsr_cpc_base) {
 		fsl_dp_priv.ccsr_cpc_base =
-			fsl_of_iomap("fsl,t1023-l3-cache-controller");
+			fsl_of_iomap("fsl,t1024-l3-cache-controller");
 		if (!fsl_dp_priv.ccsr_cpc_base)
 			goto err;
 	}
-- 
2.9.3

